pkgs/applications/networking/browsers/ladybird/default.nix

```diff
@@ -49,6 +53,17 @@ in gcc11Stdenv.mkDerivation {
     "-DENABLE_UNICODE_DATABASE_DOWNLOAD=false"
   ];
 
+  # error: use of undeclared identifier 'aligned_alloc'
+  NIX_CFLAGS_COMPILE = toString (lib.optionals (stdenv.isDarwin && lib.versionOlder stdenv.targetPlatform.darwinSdkVersion "11.0") [
+    "-include mm_malloc.h"
+    "-Daligned_alloc=_mm_malloc"
+  ]);
```

Comment:
Why not use the 11 sdk for both darwin platforms?

Reply:
It seemed (perhaps incorrectly) to me that `llvmPackages_14.stdenv` was pulling `darwin.apple_sdk` stuffs. Moved to #203532 as the fix gets complicated when adding aarch64-linux support.